docs/data/data-grid/custom-columns/EditingWithDatePickers.tsx

```diff
@@ -81,7 +65,20 @@ function GridEditDateCell({
       value={value}
       autoFocus
       onChange={handleChange}
-      slots={{ textField: WrappedGridEditDateInput }}
+      slotProps={{
+        textField: {
+          variant: 'standard',
```

Comment:
Looks like it's doable with `slotProps`, isn't it smoother and preferred over slots overriding when possible?

Reply:
Hi @LukasTy , I've tried relying on accessible DOM structure here https://github.com/mui/mui-x/pull/15967/commits/f6e6eeee4372dd576cde277ad1acd726d5eb2f40# but ran into accessibility warning when input receives focus, Does https://github.com/mui/mui-x/pull/15985 solves that?

Reply:
@k-rajat19 Yes, that change aims to solve the exact "problem" of calling `inputRef.focus()` and not seeing any effect, because the `input` is a hidden element aimed to store the bound value on the accessible DOM.
We'll see if there are no objections to that approach, given that it could be seen as somewhat "hacky". 🙈 
___
IMHO, we should keep using the `autoFocus` prop for that particular demo to minimize the code.
If we want to demo `editMode="row"`, the `autoFocus` should be removed as no explicit field focusing is needed in this mode.